ansible_collections/arista/avd/roles/eos_designs/doc/fabric-variables.md

```diff
@@ -31,6 +31,17 @@ underlay_ospf_bfd_enable: < true | false | default -> false >
 
 # Underlay ISIS | Required when < underlay_routing_protocol > == ISIS
 isis_area_id: < isis area | default -> "49.0001" >
+isis_default_is_type: < level-1-2, | level-1 | level-2 | Default -> level-2 >
+# ISIS Parameters applied to backbone interfaces
```

Comment:
Add a blank line above the comment.

Reply:
actually just move these vars that are only applicable to backbone_interfaces for now, to the backbone_interfaces doc

Reply:
Moved to the backbone-interface.md doc.